pkg/controller/controller.go

```diff
@@ -298,3 +299,20 @@ func (p *csiProvisioner) volumeIdToHandle(id string) string {
 func (p *csiProvisioner) volumeHandleToId(handle string) string {
 	return handle
 }
+
+func getCredentialsFromSecret(k8s kubernetes.Interface, sn, ns string) map[string]string {
```

Comment:
might be worth having longer variables names in function interface. I know short Go variable names are idiomatic but these are not common abbreviations and are almost too similar to each other.

Reply:
getCSICredsFromSecrets would work fo you?

Reply:
sorry I was referring to "sn" and "ns" as being too short

Reply:
Got it, sorry, done..